### DISCUSSION TITLE
How do I turn a datetime.datetime into a BeautifulDate?

### DISCUSSION THREAD

**Discussion by N-Demir** [kuzmoyev/beautiful-date#25] [Q&A] [answered]
Wondering because I realize
<img width="364" alt="Screenshot 2023-02-24 at 8 41 23 PM" src="https://user-images.githubusercontent.com/23620644/221338591-65afb633-e860-426f-8bf2-ba2a80cbe005.png">

which doesn't actually do what I would expect it to do.

**Comment by kuzmoyev** [comment, marked as answer]
`BeautifulDate` can only represent "date" object, not datetime. (E.x. try `D.today()`, that'll give you `BeautifulDate`).

But your issue is that `+ 1 * day` sets the `day` to 1. 
 I assume you wanted to add 1 day, so you want to use `+ 1 * days` (note the plural, check [the note in the docs](https://github.com/kuzmoyev/beautiful-date#addingsubtractingsetting-timedeltas)). 

I understand the confusion for this particular case when you're adding 1 days :)

**Reply by kuzmoyev** [reply]
It follows the format of [dateuitil](https://dateutil.readthedocs.io/en/stable/relativedelta.html#module-dateutil.relativedelta)

**Reply by N-Demir** [reply]
Wow thank you! I totally would have missed this nuance

**Reply by kuzmoyev** [reply]
I should probably add this example to the docs 😊
